### PR TITLE
Aluminum's atomicNumber is 13.

### PR DESCRIPTION
**Pull request by sunlee334**
added which atomic number is.

### CODE REVIEW

File: is.js [hunk 1]
```diff
@@ -129,6 +129,11 @@ var is = function is(x) {
             return parseInt(x.toString().split("").reverse().join("")) == THIRTEEN;
           }
         },
+        atomicNumber: {
+          thirteen: function() {
+            return x.toLowerCase().includes("a","l","u","m","i","u","m");
```
Comment (mkem114) [inline is.js]: Doesn't include the n in "aluminum" nor does it account for "aluminium"

Comment (elisalter) [reply is.js]: @mkem114 
`return x.toLowerCase().includes("a", "l", "u", Math.min("i", "u", "m"));`

Above line appears to be foolproof, it includes the "a", "l", "u" Math.min and "i", "u", "m".

Comment (sunlee334) [reply is.js]: sry, now it include the 'n'.

Comment (sunlee334) [reply is.js]: > sry, now it include the 'n'.
@mkem114 @elisalter @jezen

Comment (jezen) [reply is.js]: @sunlee334 Fanttttttttassstic effort my friend. Thamk you.

**Comment by illegitimate-egg** [comment]
This should allow aluminium as well

**Pull request merged**